README.md

```diff
@@ -12,23 +14,24 @@ This repo aims to provide a stripped down version of the default [WooCommerce E2
   * **Mac** - https://docs.docker.com/docker-for-mac/install/
   * **Windows** - https://docs.docker.com/docker-for-windows/install/
 
```

Comment:
Can we also add `NVM` as a pre-requisite. Could be an optional one as well mentioning that the tests make use of Node v12

Reply:
> Since we added .nvmrc, this line can be removed.

We should leave that for the folks who are following the instructions while implementing in an existing project.

> Can we also add NVM as a pre-requisite.

It is in the pre-requisite list.